packages/@sanity/components/src/utilities/tryFindScrollContainer.js

```diff
@@ -0,0 +1,31 @@
+/* global window, document */
+export default function tryFindScrollContainer(element, callback) {
```

Comment:
Why a callback here? It doesn't do anything async?

Reply:
👍